learning-progression.md

```diff
@@ -132,54 +132,97 @@ basic.showIcon(IconNames.Butterfly)
 
 ### Step 5: Conditionals
 
+#### 1. Study
+
+In the English language, a statement that begins with "if" is followed by a condition. If the condition is met, the next statement follows as a logical consequence. For example, in the sentence "If it starts to rain, then I will get wet." The condition is that "it starts to rain." When the condition becomes true, the second part of the sentence - "I will get wet" - also becomes true.
+
+In javascript, we use statements that begin with `if` in the same way. Take a look at the following code:
+
 ```javascript
-if (true) {
+if (true) { //This condition is always true, so the code in this block will run.
     basic.showIcon(IconNames.Heart)
     basic.pause(2000)
     basic.showIcon(IconNames.Butterfly)
-} else {
+} else { //Everything in this block is ignored because the first "if" statement was true.
     basic.showIcon(IconNames.Angry)
     basic.pause(2000)
     basic.showIcon(IconNames.Snake)
 }
-   
-if (false) {
+```
+
+In the above example, the statement begins with `if`, and the condition written within the parentheses is `true`. Therefore, the statements inside the first block of code are run. The `else` block can be thought of as an alternative to the `if` statement - it will only run if the condition for `if` is not met. Therefore, in this version of our code it will be ignored.
+
+_In javascript, words like `true` and `false` are special keywords reserved by the language. These keywords can only be used to represent a specific boolean value. More on booleans in the next step._
+
+Conversely, if the condition provided within the parentheses is `false`, then the statements in the first block of code will be skipped. Instead, the code in the block following `else` will be run.
+
+```javascript
+if (false) { //This block will be ignored.
     basic.showIcon(IconNames.Heart)
     basic.pause(2000)
     basic.showIcon(IconNames.Butterfly)
-} else {
+} else { //The only condition for this block of code to run, is that the above block did not run.
     basic.showIcon(IconNames.Angry)
     basic.pause(2000)
     basic.showIcon(IconNames.Snake)
 }
 ```
 
-#### 1. Study
+Code written as "`if`-`else`" statements is always executed in an either-or fashion. This means that only one of the two blocks will run. So the program will split and only one of the two possible outcomes will be executed, this is known as a conditional "branch" in programming. 
 
 #### 2. Apply
 
+1. Write a program that shows a heart icon if the condition is true and a butterfly if the condition is false.  
```

Comment:
Do you expect the literals or any condition that evaluates to true or false. If the former, backtick the literals; if the latter, you need to explain more about conditional statements (comparisons) in the Study section.